src/main/java/com/yurjinia/project_structure/project/service/ProjectService.java

```diff
@@ -75,6 +77,20 @@ private void validateIfProjectExists(ProjectDTO projectDTO) {
         }
     }
 
+    private void associateUserWithProject(String userEmail, ProjectEntity projectEntity) {
+        UserEntity userEntity = userService.getByEmail(userEmail);
+        List<UserEntity> userEntityList = new ArrayList<>();
+        userEntityList.add(userEntity);
+        projectEntity.setUsers(userEntityList);
```

Comment:
Recently we had a discussion together with Pavlo about what's the easier approach of solving issues in case we an entity and it has a collection of entities and it has `null` value.

You need to assign it directly.

UserEntity {
    Set<ProjectEntity> projects = new HashSet<>();
}